spec/rubocop/cop/style/braces_around_hash_parameters_spec.rb

```diff
@@ -71,7 +71,7 @@
     it 'registers an offense for one hash parameter with braces and ' \
        'whitespace' do
       expect_offense(<<-RUBY.strip_indent)
-        where(  
+        where(
```

Comment:
I believe there is supposed to be trailing whitespace here. It was added in aab0516dd34247aed2588254690b5639524684ca

Reply:
Fixed 😊